packages/custom-field/theme/lumo/vaadin-custom-field-styles.js

```diff
@@ -83,6 +83,11 @@ const customField = css`
   :host([theme~='small'][has-label]) [part='error-message'] {
     font-size: var(--lumo-font-size-xxs);
   }
+
+  /* When custom-field is used with non-optimized components like rich-text-editor */
+  :host([theme~='whitespace'][has-label]) [part='label'] {
```

Comment:
Not super happy with the `whitespace` name yet, maybe we can do a quick brainstorming in Slack for alternatives.

Reply:
Feels like a silly variant anyways, so I would not worry about the name too much 😅

Reply:
Ping @sissbruecker, done